data/maintainers.ts

```diff
@@ -0,0 +1,23 @@
+export interface IMaintainer {
```

Comment:
The name `IMaintainer` stands for `Interface Maintainer` or something similar, right?

Reply:
Yes, I was thinking we should probably update it so all interfaces are consistent